gputop-server/gputop-server.c

```diff
@@ -973,6 +972,71 @@ gputop_get_cmd_line_pid(uint32_t pid, char *buf, int len)
     return res;
 }
 
+void  handle_update_hw_id_map(uint64_t *vgpu_id,
+                             uint64_t *ctx_hw_id,
+                             int *current_vgpu_num)
+{
+  int i = 0;
+  int UUID_length = 36;
+  DIR *vgpu_dir;
+  char *vgpu_path="/sys/bus/pci/devices/0000:00:02.0";
+  char *vgpu_id_path, *hw_id_path;
+  struct dirent *entry;
+  bool success_vgpu_id, success_hw_id;
+
+  vgpu_dir=opendir(vgpu_path);
+  if (vgpu_dir == NULL) {
+        fprintf(stderr, "The path %s doesn't exist!\n", vgpu_path);
+        return;
+  }
+
+  while (entry = readdir(vgpu_dir)) {
+      if (entry->d_type == DT_DIR && (strlen(entry->d_name)==UUID_length)) {
+      int ret_vgpu_id_path = asprintf(&vgpu_id_path, "%s/%s/intel_vgpu/vgpu_id", vgpu_path, entry->d_name);
```

Comment:
Indentation is wrong here.
You seem to mix 2 different spacing in your change (2 vs 4 spaces), please be consistent with the rest of the file. I believe it's 4 spaces.

Reply:
Sorry for my mistake, I have corrected it.